lib/smart_answer_flows/next-steps-for-your-business.rb

```diff
@@ -0,0 +1,133 @@
+# ======================================================================
+# The flow logic.
+# ======================================================================
+
+module SmartAnswer
+  class NextStepsForYourBusinessFlow < Flow
+    def define
+      name "next-steps-for-your-business"
+      start_page_content_id "4d7751b5-d860-4812-aa36-5b8c57253ff2"
+      flow_content_id "981e0708-9fa5-42fb-baf5-ee5630a9b722"
+      status :draft
+      use_session true
+      # ======================================================================
+      # Available input types:
+      # ======================================================================
+      # - Checkbox
+      # - Country select
+      # - Date
+      # - Money
+      # - Radio
+      # - Postcode
+      # - Salary
+      # - Value (text)
+
+      # ======================================================================
+      # What is your company registration number?
+      # ======================================================================
+      value_question :crn? do
+        on_response do |response|
+          self.calculator = Calculators::NextStepsForYourBusinessCalculator.new
+          calculator.crn = response
+        end
+
+        next_node do
+          question :annual_turnover?
+        end
+      end
+
+      # ======================================================================
+      # Will your business take more than £85,000 in a 12 month period?
+      # ======================================================================
+      radio :annual_turnover? do
+        option :more_than_85k
+        option :less_than_85k
+        option :maybe_85k
+
+        on_response do |response|
+          calculator.annual_turnover = response
+        end
+
+        next_node do
+          question :employ_someone?
+        end
+      end
+
+      # ======================================================================
+      # Do you want to employ someone?
+      # ======================================================================
+      radio :employ_someone? do
+        option :will_employ
+        option :already_employ
+        option :no_employ
+        option :maybe_employ
```

Comment:
Nit: This might be more concise

```suggestion
        option :yes
        option :in_future
        option :no
        option :not_sure
```